test/Unit/Porter/Net/Caching/CachingConnectorTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+namespace ScriptFUSIONTest\Unit\Porter\Net\Caching;
+
+use Mockery\Adapter\Phpunit\MockeryPHPUnitIntegration;
+use ScriptFUSION\Porter\Cache\MemoryCache;
+
+final class CachingConnectorTest extends \PHPUnit_Framework_TestCase
+{
+    use MockeryPHPUnitIntegration;
```

Comment:
It seems we're not using Mockery here (although perhaps we should be?) As is, this can be removed.